source/vir/src/ast_visitor.rs

```diff
@@ -240,6 +240,10 @@ where
                         }
                     }
                 }
+                ExprX::OpenInvariant(inv, _binder, body) => {
+                    expr_visitor_control_flow!(expr_visitor_dfs(inv, map, mf));
+                    expr_visitor_control_flow!(expr_visitor_dfs(body, map, mf));
```

Comment:
My understanding is that the `binder` is relevant to the `body` (it's the "inner" introduced by `open_invariant!(&i => inner => {`), so I think this should be:

```
ExprX::OpenInvariant(inv, binder, body) => {
    expr_visitor_control_flow!(expr_visitor_dfs(inv, map, mf));
    map.push_scope(true);
    let _ = map.insert(binder.name.clone(), binder.a.clone());
    expr_visitor_control_flow!(expr_visitor_dfs(body, map, mf));
    map.pop_scope();
}
```

Reply:
Similarly to https://github.com/secure-foundations/verus/blob/cc89012c5c73be6cb2493259e83854f088efd30f/source/vir/src/ast_visitor.rs#L434-L442